ptf/saiport.py

```diff
@@ -0,0 +1,2829 @@
+        ''' Tests port qos pfc priority to priority group map attribute '''
+        print("portQosPfcPriorityToPriorityGroupMapAttributeTest")
+        # SAI_PORT_ATTR_QOS_PFC_PRIORITY_TO_PRIORITY_GROUP_MAP
+        #  - qos_pfc_priority_to_priority_group_map
+        # create/set
+        ingress_key_list = [1]
+        ingress_data_list = [0]
+        qos_map_id = create_qos_map(
+            self.client,
+            SAI_QOS_MAP_TYPE_PFC_PRIORITY_TO_PRIORITY_GROUP,
+            ingress_key_list,
+            ingress_data_list)
+        self.assertTrue(qos_map_id != 0,
+                        "Failed to create qos_map")
+        try:
+            # create/get
+            qos_pfc_ppg_map = None
+            initial_qos_map_id = 0
+            if TEST_QOS_MAP_ON_CREATE_PORT is True:
+                initial_qos_map_id = qos_map_id
+
+            sai_list = sai_thrift_u32_list_t(count=1, uint32list=[34])
+            self.portx = sai_thrift_create_port(
+                self.client,
+                hw_lane_list=sai_list,
+                speed=TEST_DEFAULT_SPEED,
+                qos_default_tc=TEST_QOS_DEFAULT_TC,
+                qos_pfc_priority_to_priority_group_map=qos_pfc_ppg_map)
+            self.assertTrue(self.portx != 0, "Failed to create port")
+
+            attr = sai_thrift_get_port_attribute(
+                self.client, self.portx,
+                qos_pfc_priority_to_priority_group_map=True)
+            self.assertEqual(attr['qos_pfc_priority_to_priority_group_map'],
+                             initial_qos_map_id)
+
+            if TEST_QOS_MAP_ON_CREATE_PORT is True:
+                status = sai_thrift_set_port_attribute(
+                    self.client, self.portx,
+                    qos_pfc_priority_to_priority_group_map=0)
+                self.assertEqual(status, SAI_STATUS_SUCCESS,
+                                 "Failed to set port attribute")
+
+            # set/get
+            status = sai_thrift_set_port_attribute(
+                self.client,
+                self.portx,
+                qos_pfc_priority_to_priority_group_map=qos_map_id)
+            self.assertEqual(status, SAI_STATUS_SUCCESS,
+                             "Failed to set port attribute")
+
+            attr = sai_thrift_get_port_attribute(
+                self.client, self.portx,
+                qos_pfc_priority_to_priority_group_map=True)
+            self.assertEqual(attr['qos_pfc_priority_to_priority_group_map'],
+                             qos_map_id)
+
+        finally:
+            # Remove the qos_map
+            status = sai_thrift_set_port_attribute(
+                self.client, self.portx,
+                qos_pfc_priority_to_priority_group_map=0)
+            status = sai_thrift_remove_qos_map(self.client, qos_map_id)
+            status = sai_thrift_remove_port(self.client, self.portx)
+
+
+@group("draft")
+class SAIBridgePortNoFloodTest(SaiHelperBase):
```

Comment:
Shouldn't this move to saibridge_port.py

Reply:
I agree, we should move both of them. I have already PR for bridgeport module opened in OCP repo. I will add these tests to it so please remove them from this file.

Reply:
Thanks, addressed in next commit.